PROJECT_PROGRESSION.md

```diff
@@ -247,27 +255,98 @@ A project's progress toward its growth plan goals will be reviewed on a regular
 
 Projects are generally expected to meet their growth plan goals within two years.
 
-## V. Sunsetting Checklist
+## V. End of Life Project Checklist
 
-<!--If this checklist is updated please ensure `https://github.com/openjs-foundation/project-status/blob/HEAD/.github/ISSUE_TEMPLATE/02-project-sunsetting-checklist-template.md` is updated as well -->
+Projects that are transitioning to End of Life should follow this checklist. Tasks will be completed together with the CPC and Foundation staff. The goal is to make sure that the Foundation is able to make required changes to the project if the need arises, and revive the project should this become desirable. If you have any questions or need help, the OpenJS Foundation CPC is available to assist.
+
+### Prepare Project For End of Life
```

Comment:
Should this be moved to a dedicated file as we have for incubating/onboarding checklist?